### PR TITLE
[issue-2] Fix localized images path

### PR DESCRIPTION
**Pull request by rxu**
Localized images should be located in styles/{style_name}/theme/{iso}/ folder
rather than in styles/{style_name}/imageset/{iso}/ for Ascraeus
since there's no imageset in phpBB 3.1.x.

### CODE REVIEW

File: guidelines.md [hunk 1]
```diff
@@ -132,7 +132,7 @@ These files must be placed in the `language/{iso}/` directory, next to the `iso.
                     viewtopic.php
               styles/
                  prosilver/
-                    imageset/
+                    theme/
                        {iso}/
                           button_pm_forward.gif
```
Comment (nickvergessen) [inline guidelines.md]: buttons have been removed

File: guidelines.md [hunk 2]
```diff
@@ -147,7 +147,7 @@ These files must be placed in the `language/{iso}/` directory, next to the `iso.
                           imageset.cfg
```
Comment (nickvergessen) [inline guidelines.md]: files does not exist anymore, but a stylesheet.css?

Comment (rxu) [inline guidelines.md]: Indeed.

**Comment by rxu** [comment]
Updated #3.

File: guidelines.md [hunk 3]
```diff
@@ -132,22 +132,15 @@ These files must be placed in the `language/{iso}/` directory, next to the `iso.
...
-                          button_pm_reply.gif
-                          button_topic_locked.gif
-                          button_topic_new.gif
-                          button_topic_reply.gif
                           icon_contact_pm.gif
                           icon_post_edit.gif
                           icon_post_quote.gif
                           icon_user_online.gif
-                          imageset.cfg
-                          index.htm
```
Comment (nickvergessen) [inline guidelines.md:148]: https://github.com/phpbb/phpbb-translation-policies/blob/develop-olympus/validation.md#indexhtm
Can you add `index.htm (optional)` to all folders please?

**Comment by rxu** [comment]
Updated #3.

**Pull request merged**